File: Wirtualny Swiat/PiotrPrzymus193736JavaWS/Owca.java
package WSJAVA;

public class Owca extends Zwierze{
    public Owca(int polX, int polY, Swiat swiat) {
        super(polX, polY, swiat);
        this.sila = 4;
        this.inicjatywa = 4;
        this.znak_gatunku = Stale.OWCA;
        this.wiek = 0;
        this.nazwa = "Owca";
    }
    public Owca(int polozenieX, int polozenieY, int sila, int inicjatywa, char znak_gatunku, int wiek, String nazwa, Swiat swiat) {
        super(polozenieX, polozenieY, sila, inicjatywa, znak_gatunku, wiek, nazwa, swiat);
    }

    @Override
    public void PostawDziecko(int x, int y)
    {
        swiat.DodajOrganizm(new Owca(x, y, swiat));
        //swiat.gotoxy(40, 8);
    }
}

File: Wirtualny Swiat/Piotr Przymus 193736 Wirtualny Swiat/Owca.cpp
#include "Owca.h"

Owca::Owca(int polX, int polY, Swiat& swiat)
	: Zwierze(polX, polY, swiat)
{
	wiek = 0;
	inicjatywa = 4;
	sila = 4;
	znak_gatunku = OWCA;
	nazwa = "Owca";
}

Owca::Owca(int polozenieX, int polozenieY, int sila, int inicjatywa, char znak, int wiek, string nazwa, Swiat& swiat)
	: Zwierze(polozenieX, polozenieY, sila, inicjatywa, znak, wiek, nazwa, swiat)
{
}

Owca::~Owca()
{
}

void Owca::PostawDziecko(int x, int y)
{
	swiat.DodajOrganizm(new Owca(x, y, swiat));
	swiat.gotoxy(40, 8);
}